controllers/glanceapi_controller.go

```diff
@@ -542,11 +542,14 @@ func (r *GlanceAPIReconciler) generateServiceConfigMaps(
 		templateParameters["OsdCaps"] = ceph.GetOsdCaps(instance.Spec.CephBackend.Pools)
 	}
 
-	// We set "show_image_direct_url" to true if this is an internal GlanceAPI,
-	// otherwise we default to false
-	templateParameters["ShowImageDirectUrl"] = false
+	// Configure the internal GlanceAPI to provide image location data, and the
+	// external version to *not* provide it.
 	if instance.Spec.APIType == glancev1.APIInternal {
 		templateParameters["ShowImageDirectUrl"] = true
+		templateParameters["ShowMultipleLocations"] = true
```

Comment:
Since we want both to have the same value we could just name it `"IsInternalService"` or something like that and use a single template variable.

And we could then just simplify L547 to L552 to:

```
templateParameters["IsInteralService"] = instance.Spec.APIType == glancev1.APIInternal
```